File: rb/test/exists_test.rb
# SlaUptimeCalculator SDK exists test

require "minitest/autorun"
require_relative "../SlaUptimeCalculator_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = SlaUptimeCalculatorSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- SlaUptimeCalculator SDK exists test

local sdk = require("sla-uptime-calculator_sdk")

describe("SlaUptimeCalculatorSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
